src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -20,7 +20,14 @@
      * <p>Пример: Madam, I'm Adam!</p>
      * <p>Результат: true</p>
      */
+
     public boolean isPalindrome(String text) {
+        String cleanText = text.replaceAll("[^a-zA-Z\\d]", "");
+        StringBuilder sb = new StringBuilder(cleanText);
+
+        if (cleanText.toLowerCase().equals(sb.reverse().toString().toLowerCase())) {
```

Comment:
let's try to solve it without if construction

Reply:
done